src/java/org/apache/cassandra/db/virtual/AccordVirtualTables.java

```diff
@@ -73,6 +79,43 @@ public DataSet data()
         }
     }
 
+    public static final class CommandStoreCache extends AbstractVirtualTable
+    {
+        private CommandStoreCache(String keyspace)
+        {
+            super(parse(keyspace,
+                        "Accord Command Store Cache Metrics",
+                        "CREATE TABLE accord_command_store_cache(\n" +
+                        "  id int,\n" +
+                        "  queries bigint,\n" +
```

Comment:
one issue with this table is it uses the global stats and not the instance stats... I kinda feel we should expose both?

What I see in practice is that `commands` and `commandsForKeys` have very different cache hit stats and the global is showing it for all...  If you look at the patch I did to add JFR support I show the stats per instance with a label so we know where its coming from... that helps show if commands are getting flushed out constantly in favor of CFK

Reply:
I think we've now discussed this bit offline. For posterity, mentioned in https://issues.apache.org/jira/browse/CASSANDRA-18732?focusedCommentId=17848391&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-17848391